definitions/npm/@testing-library/user-event_v13.x.x/flow_v0.104.x-/user-event_v13.x.x.js

```diff
@@ -0,0 +1,170 @@
+declare type TypeOpts = {|
+  skipClick?: boolean,
+  skipAutoClose?: boolean,
+  initialSelectionStart?: number,
+  initialSelectionEnd?: number,
+|};
+declare type TypeOptsDelay = {|
+  skipClick?: boolean,
+  skipAutoClose?: boolean,
+  delay: number,
+  initialSelectionStart?: number,
+  initialSelectionEnd?: number,
+|};
```

Comment:
**Nit:** We could use a spread here to avoid duplication.

Reply:
Done :)